tutorials/tutorial-use-timescale-prometheus-grafana.md

```diff
@@ -0,0 +1,450 @@
+queries (one for each database):
+
+```sql
+SELECT
+  time,
+  avg(value) AS "defaultdb connections"
+FROM metrics_values
+WHERE
+  labels_id = 296
+GROUP BY 1
+ORDER BY 1;
+
+SELECT
+  time,
+  avg(value) AS "defaultdb connections"
+FROM metrics_values
+WHERE
+  labels_id = 297
+GROUP BY 1
+ORDER BY 1;
+```
+
+Our result for the first of the above queries should look something like this:
+
+```bash
+           time            | defaultdb connections 
+----------------------------+-----------------------
+ 2020-03-16 04:11:27.936+00 |                     2
+ 2020-03-16 04:11:37.937+00 |                     2
+ 2020-03-16 04:11:47.937+00 |                     2
+```
+
+Depending on your installation of Prometheus, the `metric_labels` table may vary. 
+Therefore, it’s probably better not to hardcode the `label_id` into your SQL query 
+and instead `JOIN` queries on the two tables, like this:
+
+```sql
+SELECT
+  time,
+  avg(value) AS "defaultdb connections"
+FROM metrics_values
+JOIN metrics_labels ON metrics_values.labels_id = metrics_labels.id
+WHERE
+  metrics_labels.metric_name LIKE 'postgresql_pg_stat_activity_conn_count'
+GROUP BY 1
+ORDER BY 1;
+```
+
+We can visualize this query in Grafana using a Graph visualization by selecting 
+the ‘Edit SQL’ button in the query builder and entering the following query, which 
+looks similar to the raw SQL query from earlier but is altered to support Grafana:
+
+```sql
+SELECT
+  $__timeGroupAlias("time", 1m),
+  avg(value) AS "defaultdb connections"
+FROM metrics_values
+JOIN metrics_labels ON metrics_values.labels_id = metrics_labels.id
+WHERE
+  $__timeFilter("time") AND
+  metrics_labels.metric_name LIKE 'postgresql_pg_stat_activity_conn_count'
+GROUP BY 1
+ORDER BY 1;
+```
+
+The result of your query should look like this:
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/screenshots-for-prometheus-tutorial/grafana_query_1_screenshot.png" alt="Visualizing PostgreSQL connections in Grafana"/>
+
+### What is the maximum and average percent of memory usage?
+
+Knowing the maximum and average memory usage of your infrastructure gives 
+you an idea of when you are about to require an upgrade to your service plan 
+or instance size.
+
+The problem with obtaining this information is that running this query over _all_ 
+of your data several times a day can take a long time. TimescaleDB includes a 
+feature called [continuous aggregation][continuous-aggregates]. A continuous aggregate 
+recomputes a query automatically at user-specified time intervals and stores the results 
+in a table. Thus, instead of everyone running an aggregation query each time, the 
+database can run a common aggregation periodically in the background, and users can 
+query the results of the aggregation. Continuous aggregates should improve database 
+performance and query speed for common calculations.
+
+In our case, we want to maintain a continuous aggregation for the average, maximum, and 
```

Comment:
```suggestion
In our case, we want to maintain a continuous aggregate for the average, maximum, and 
```
I'm never sure what the proper term for the aggregate itself is, but I _think_ it's 'continuous aggregate'?